controllers/exclude_instances.go

```diff
@@ -40,62 +40,54 @@ func (e ExcludeInstances) Reconcile(r *FoundationDBClusterReconciler, context ct
 	}
 	defer adminClient.Close()
 
-	removalCount := 0
-	for _, processGroup := range cluster.Status.ProcessGroups {
-		if processGroup.Remove {
-			removalCount++
-		}
+	addresses := make([]string, 0)
+	missingProcesses := make([]string, 0)
+
+	exclusions, err := adminClient.GetExclusions()
```

Comment:
Are we sure we want to check the exclusion list on every reconciliation, even in the common case where it won't be used?